src/components/Settings/Settings.tsx

```diff
@@ -68,7 +69,16 @@ const SettingsModal: React.FC<ModalProps> = ({
           >
             Hardware
           </div>
-
+          <div
+            className={`flex items-center rounded cursor-pointer p-2 border-b border-gray-200 hover:bg-gray-600 text-sm ${
+              activeTab === "textGeneration"
```

Comment:
Perhaps it would be a good idea to extract the tab names out into an enum, to avoid mispellings, and in the future if we wanted to reference this, it would be fairly easy.